validation/src/validation_service/mod.rs

```diff
@@ -236,6 +232,57 @@ impl<C, N, P, SC, SP> ServiceBuilder<C, N, P, SC, SP> where
 	}
 }
 
+/// Abstraction over `collation_fetch`.
+pub trait CollationFetch {
```

Comment:
`CollationFetch` is already a wrapper around the `Collators` abstraction. I say we only need one or the other

Reply:
I'd suggest this to be non-public, if it's only used for mocking

Reply:
This doesn't need to be `pub`, does it? The user of the trait is `pub(crate)` only